### PR TITLE
Serverless ebook

### PR DESCRIPTION
**Pull request by BillWagner**
Final PR to make the serverless eBook live.

/cc @JeremyLikness

### CODE REVIEW

**Comment by mairaw** [comment]
I'm doing one quick last edit pass directly on the branch and will push the commit soon. But I'll post some questions here.

File: docs/standard/serverless-architecture/orchestration-patterns.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+---
+# Orchestration Patterns
+
+Durable Functions makes it easier to create stateful workflows that are composed of discrete, long running activities in a serverless environment. Since Durable Functions is able to track the progress of your workflows and periodically checkpoints the execution history, it lends itself to implementing some interesting patterns.
+
+## Function chaining
+
+In a typical sequential process, activities need to execute one after the other in a particular order. Optionally, the upcoming activity may require some output from the previous function. This dependency on the ordering of activities creates a function chain of execution.
+
+The benefit of using Durable Functions to implement this workflow pattern comes from its ability to do checkpointing. If the server crashes, the network times out or some other issue occurs, Durable functions is able to resume from the last known state and continue running your workflow event if it's on another server.
```
Comment (mairaw) [inline docs/standard/serverless-architecture/orchestration-patterns.md]: workflow event if it's on another server -> should event be even?

**Review by mairaw — APPROVED** [review]
LGTM. Would be good to check my last question before merging.

File: docs/standard/serverless-architecture/orchestration-patterns.md (hunk 1 above)
Comment (BillWagner) [reply docs/standard/serverless-architecture/orchestration-patterns.md]: /cc @JeremyLikness Can you comment on @mairaw 's question? (Once we have that, I'll merge).

**Pull request merged**